internal/httphandler/ledger_request_handler.go

```diff
@@ -36,14 +36,18 @@ func NewLedgerRequestHandler(db bcdb.DB, logger *logger.SugarLogger) http.Handle
 	handler.router.HandleFunc(constants.GetBlockHeader, handler.blockQuery).Methods(http.MethodGet)
 	// HTTP GET "/ledger/path?start={startId}&end={endId}" gets shortest path between blocks
 	handler.router.HandleFunc(constants.GetPath, handler.pathQuery).Methods(http.MethodGet).Queries("start", "{startId:[0-9]+}", "end", "{endId:[0-9]+}")
-	// HTTP GET "/ledger/proof/{blockId}?idx={idx}" gets proof for tx with idx index inside block blockId
+	// HTTP GET "/ledger/proof/tx/{blockId}?idx={idx}" gets proof for tx with idx index inside block blockId
 	handler.router.HandleFunc(constants.GetTxProof, handler.txProof).Methods(http.MethodGet).Queries("idx", "{idx:[0-9]+}")
+	// HTTP GET "/ledger/proof/data/{blockId}/{dbname}/{key}?deleted={true|false}" gets proof for tx with idx index inside block blockId
+	handler.router.HandleFunc(constants.GetDataProof, handler.dataProof).Methods(http.MethodGet).Queries("deleted", "{deleted:true|false}")
 	// HTTP GET "/ledger/tx/receipt/{txId}" gets transaction receipt
 	handler.router.HandleFunc(constants.GetTxReceipt, handler.txReceipt).Methods(http.MethodGet)
 	// HTTP GET "/ledger/path?start={startId}&end={endId}" with invalid query params
 	handler.router.HandleFunc(constants.GetPath, handler.invalidPathQuery).Methods(http.MethodGet)
-	// HTTP GET "/ledger/proof/{blockId}?idx={idx}" with invalid query params
+	// HTTP GET "/ledger/proof/tx/{blockId}?idx={idx}" with invalid query params
```

Comment:
This will break SDK, need to reflect it there as well

Reply:
Sure, next step will update sdk. Lets tag it after that to 0.1.1

Reply:
Because sdk call to same `constants.URLTxProof(blockNum, txIndex)` function, new tag after merge will be enough. In any case, opened new issue to update sdk: https://github.com/IBM-Blockchain/bcdb-sdk/issues/29